tools/devtool

```diff
@@ -714,7 +722,7 @@ cmd_test() {
     cmd_fix_perms
 
     # undo performance tweaks (in case the instance gets recycled for a non-perf test)
-    if [[ "$BUILDKITE" == "true" ]] && [[ "$(uname --machine)" == "x86_64" ]]; then
+    if [[ $performance_tweaks -eq 1 ]] && [[ "$BUILDKITE" == "true" ]] && [[ "$(uname --machine)" == "x86_64" ]]; then
```

Comment:
hmm I think if we have the option then we would want to apply it regardless of the `BUILDKITE` value. For example if we are running it by hand.

Reply:
Yeah, originally I wanted to have the `$BUILDKITE` check here as a failsafe against running this on a laptop and then leaving the power configuration messed up, but it does also rule out EC2 instances... Now that we also have the unapply working correctly, I feel a bit better about not having this check, so can remove it :)

Reply:
Done